src/main/java/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategy.java

```diff
@@ -29,8 +29,6 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import javax.servlet.ServletException;
-

```

Comment:
Many unused imports in this plugin...
My Eclipse doesn't warn them for https://groups.google.com/forum/#!topic/jenkinsci-dev/4rvorfnry1w .